libphal/phal_dump.C

```diff
@@ -416,9 +468,14 @@ void collectLocalRegDump(struct pdbg_target* target,
 		hwpName = "p10_sbe_localreg_dump";
 		fapiRc = p10_sbe_localreg_dump(target, true, sbeScomRegValue);
 	}
+	std::string dumpFilename = baseFilename + hwpName;
```

Comment:
The same lines of code repeated 4 times. Please move this common code to a new method in one commit and update the method in the next commit with your writeDumpFileOnChipOpFailure